test/api-digester/Outputs/stability-stdlib-source-x86_64.swift.expected

```diff
@@ -1,10 +1,13 @@
-Accessor MutableCollection.subscript(_:).Get() has generic signature change from <Self where Self : Swift.MutableCollection> to <Self where Self : Swift.MutableCollection, Self.SubSequence == Swift.Slice<Self>>
-Accessor MutableCollection.subscript(_:).Set() has generic signature change from <Self where Self : Swift.MutableCollection> to <Self where Self : Swift.MutableCollection, Self.SubSequence == Swift.Slice<Self>>
+Accessor MutableCollection.subscript(_:).Get() has parameter 0 type change from Swift.Range<Self.Index> to Swift.RangeSet<Self.Index>
+Accessor MutableCollection.subscript(_:).Get() has return type change from Swift.Slice<Self> to Swift.DiscontiguousSlice<Self>
+Accessor MutableCollection.subscript(_:).Set() has parameter 0 type change from Swift.Slice<Self> to Swift.DiscontiguousSlice<Self>
+Accessor MutableCollection.subscript(_:).Set() has parameter 1 type change from Swift.Range<Self.Index> to Swift.RangeSet<Self.Index>
```

Comment:
Yeah, unfortunately this seems to be an issue with the ABI checker rather than an actual issue. I've validated that both sets of symbols still exist in the produced dylib, so it was not a rename but rather an addition.

Reply:
Radar for the ABI checker, please.

Reply:
rdar://118989990